flocker/node/agents/cinder.py

```diff
@@ -530,7 +530,7 @@ def list_volumes(self):
         http://docs.rackspace.com/cbs/api/v1.0/cbs-devguide/content/GET_getVolumesDetail_v1__tenant_id__volumes_detail_volumes.html
         """
         flocker_volumes = []
-        for cinder_volume in self.cinder_volume_manager.list(detailed=False):
+        for cinder_volume in self.cinder_volume_manager.list(detailed=True):
```

Comment:
It'd be possible but probably more trouble than it's worth. 
At this point the cinder version detection has already taken place and the CinderBlockDeviceAPI has an object conforming to an interface that we've defined `ICinderVolumeManager`.
Since we're aiming to drop support for  CinderV1 (Rackspace) I'm going to leave it.